travis/travis_install_nightly

```diff
@@ -84,7 +84,17 @@ fi;
 
 if [ "${WKHTMLTOPDF_VERSION}" != "" ]; then
     echo "Install webkit (wkhtmltopdf) patched version ${WKHTMLTOPDF_VERSION}"
-    (cd ${HOME}/maintainer-quality-tools/travis/ && wget -qO- -t 1 --timeout=240 https://downloads.wkhtmltopdf.org/0.12/${WKHTMLTOPDF_VERSION}/wkhtmltox-${WKHTMLTOPDF_VERSION}_linux-generic-amd64.tar.xz | tar -xJ --strip-components=2 wkhtmltox/bin/wkhtmltopdf)
+
+    cd ${HOME}/maintainer-quality-tools/travis/ &&
+    wkhtmltox=$(wget -qO- -t 1 --timeout=240 https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox-${WKHTMLTOPDF_VERSION}_linux-generic-amd64.tar.xz) &&
+    WKHTMLTOX_CHECKSUM='049b2cdec9a8254f0ef8ac273afaf54f7e25459a273e27189591edc7d7cf29db'
```

Comment:
From  @moylop260 in  https://github.com/OCA/maintainer-quality-tools/pull/450#discussion_r122519817


> I'm not sure of use this environment variable just for webkit.
> I mean, we could use a wget_custom with a dictionary of checksum and url used from MQT.
> But IMHO this is other feature (then other PR).
> 
> Could we use just the change of the URL like as main comment?